exercises/knapsack/instructions.md

```diff
@@ -0,0 +1,25 @@
+# Instructions
+
+Your task is to determine which items to take to maximize their value, taking into account the knapsack's carrying capacity.
```

Comment:
```suggestion

Your task is to determine which items to take so that the total value of his selection is maximized, taking into account the knapsack's carrying capacity.
```